core/src/test/java/kafka/test/ClusterInstance.java

```diff
@@ -183,6 +193,53 @@ default Set<GroupProtocol> supportedGroupProtocols() {
 
     //---------------------------[wait]---------------------------//
 
+    @SuppressWarnings("deprecation")
+    default void verifyTopicDeletion(String topic, int partitions) throws InterruptedException {
```

Comment:
Maybe we don't need this so complicated method. It can be implemented by `waitForTopic("xxx", 0)` WDYT?